### DISCUSSION TITLE
一些任务点击取消之后，会一直是这个状态无法清除，请问这个可以怎么解决？

### DISCUSSION THREAD

**Discussion by wdj2613** [AlistGo/alist#3153] [General]
![Screenshot_2023-01-26-18-55-56-705_com android browser](https://user-images.githubusercontent.com/69485849/214824324-83b61fe8-a7d7-422f-bf19-6e67442f1440.jpg)

**Comment by ericuni** [comment]
我也遇到过这个问题, 把alist 服务重启之后OK了, 但只是一个临时的解决办法

**Reply by wdj2613** [reply]
重启解决了，多谢
